### PR TITLE
use aten convolution

### PR DESCRIPTION
**Pull request by ljfitz**
torch-mlir decomposes `aten.conv2d` as `aten.convolution`.

### CODE REVIEW

File: include/xten/Conversion/ATenToXTen.td [hunk 1]
```diff
@@ -29,8 +29,11 @@ def CheckFalse : Constraint<
 
 // Conversions
 
-def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
-          (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+// def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
+//           (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+
+def : Pat<(Torch_AtenConvolutionOp $input,$weight,$bias,$stride,$padding,$dilation,$transposed,$output_padding,$groups),
```
Comment (ljfitz) [inline include/xten/Conversion/ATenToXTen.td:36]: This still needs a check that `$transposed` is `false` and `$output_padding` is an empty list.

File: include/xten/Conversion/ATenToXTen.td [hunk 2]
```diff
@@ -29,8 +29,11 @@ def CheckFalse : Constraint<
 
 // Conversions
 
-def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
-          (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+// def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
```
Comment (flemairen6) [inline include/xten/Conversion/ATenToXTen.td]: Should we not remove that ? Or if we decide to keep it, I'd rather have it uncommented

File: lib/Transform/ATenVisualGraph.cpp [hunk 3]
```diff
@@ -693,8 +693,10 @@ struct ATenVisualGraphPass : public ATenVisualGraphBase<ATenVisualGraphPass> {
   llvm::json::Array fillProperties(Operation *op) {
     llvm::json::Array propertiesArray;
     auto props = JsonPropertiesBuilder(propertiesArray);
-    if (auto op2 = dyn_cast<Torch::AtenConv2dOp>(op)) {
-      fillPropertiesConvOp<Torch::AtenConv2dOp>(op2, std::move(props));
+    // if (auto op2 = dyn_cast<Torch::AtenConv2dOp>(op)) {
+    //   fillPropertiesConvOp<Torch::AtenConv2dOp>(op2, std::move(props));
```
Comment (flemairen6) [inline lib/Transform/ATenVisualGraph.cpp]: Same here, this can probably go away

File: lib/Transform/ATenVisualGraph.cpp [hunk 4]
```diff
@@ -785,7 +787,8 @@ struct ATenVisualGraphPass : public ATenVisualGraphBase<ATenVisualGraphPass> {
   Value getInputFromErasedPtr(Operation *op) {
     Value opInput;
 
-    if (auto op2 = dyn_cast<Torch::AtenConv2dOp>(op)) {
+    //if (auto op2 = dyn_cast<Torch::AtenConv2dOp>(op)) {
```
Comment (flemairen6) [inline lib/Transform/ATenVisualGraph.cpp]: ditto

**Review by flemairen6 — APPROVED** [review]

**Pull request merged**